feelpp/feel/feelmodels/modelproperties.cpp

```diff
@@ -136,7 +136,7 @@ ModelProperties::setup()
         if ( !M_directoryLibExpr.empty() )
             M_bc.setDirectoryLibExpr( M_directoryLibExpr );
         M_bc.setPTree( *bc );
-#if 0 // TODO
+#if 1 // TODO
```

Comment:
see https://github.com/feelpp/feelpp/commit/f43e0f3840f2916446449c5e41d8b84917b1b502#commitcomment-39984070.

An option (disable by default) can be put temporarily.

Reply:
this is now optional